vip-api/src/main/java/fr/insalyon/creatis/vip/api/SpringWebConfig.java

```diff
@@ -64,6 +68,12 @@
     private Environment env;
     private VipConfigurer vipConfigurer;
 
+    //implements rest template to send requests with tokens
+    @Bean
```

Comment:
Is this bean used somewhere ? Is it necessary for keycloak internally ?

Reply:
it is necessary to test the refresh token request